openlibrary/macros/databarWork.html

```diff
@@ -63,10 +53,10 @@
 
 $ library = None
 <style type="text/css">
-#edit-identifiers { 
```

Comment:
Consider isolating whitespace changes on their own branch or commit (so it's easier to detect changes of substance).